components/invitation_modal/__snapshots__/invitation_modal_members_step.test.jsx.snap

```diff
@@ -57,7 +62,7 @@ exports[`components/invitation_modal/InvitationModalMembersStep should match the
       className="help-text"
     >
       <FormattedMessage
-        defaultMessage="Share this link to grant member access to this team."
+        defaultMessage="Share this link to grant access to this team."
```

Comment:
Should we specify what team "this team" is? Perhaps "Share this link to invite people to the XXXXXX team."

Reply:
@eric this feels a bit redundant since we have added the team name in the header.  I recommend not taking this change.